### DISCUSSION TITLE
Customizing style

### DISCUSSION THREAD

**Discussion by ammarmol** [doccano/doccano#2202] [Q&A]
Hello,

I've been using Doccano and I find it to be a great tool for text annotation. However, I would like to customize the text style (size, color) in the labeling interface to suit my specific needs. I've looked into the documentation and explored the codebase, but I couldn't find a straightforward way to achieve this customization.

I would like to know if there is a possibility to customize the text style in Doccano's labeling interface. Specifically, I'm interested in changing the font size and color of the text that is displayed for annotation.

If this customization is possible, I would appreciate any guidance or pointers on how to achieve it. If there are any configuration options, CSS files, or components that I need to modify, please let me know. Alternatively, if this feature is not currently supported, I would like to understand the reasoning behind it.

Is there a possibility to inject HTML in the text within Doccano's annotation interface? For example, if I have a piece of text that includes an HTML table structure, is it possible to render that table within the annotation interface?

If this feature is supported, I would greatly appreciate any guidance or instructions on how to implement it. If there are any configuration options, code snippets, or specific files that need to be modified, please let me know. Additionally, if this feature is not currently available, I would like to understand the limitations or considerations that make it challenging to implement.

Any help or insights would be greatly appreciated. Thank you!

**Comment by Hironsan** [comment]
I think it's possible to change the font size and the text color. doccano mainly use Vuetify on the front end, so it should be able to change the font size and color by replacing the class.

For example, changing `title` to `body-1` should result in a smaller font size.

https://github.com/doccano/doccano/blob/f3e8c92abd9bc4ad9cb6ef67e65b38d7bede8b23/frontend/pages/projects/_id/text-classification/index.vue#L34

Reference
- [Text and typography](https://v2.vuetifyjs.com/en/styles/text-and-typography/)

**Reply by ammarmol** [reply]
Thank you very much, I have tried to modify the code, as you indicate me, I have even deleted elements but it seems that the code I modify is not the one used to display. Do you know anything about this?

I have used docker and created a new image and container with the modified code and it still doesn't show the changes.

**Comment by Hironsan** [comment]
This is just an example. Practically speaking, a menu item should be created for settings.

![](https://i.gyazo.com/b8f59e994514cec8538ef10aaf267c7e.gif)

**Comment by plegoff** [comment]
Hi,
In order to change the labels style, you could modify the file  frontend/nuxt.config.js  and change the css global section :

`css: ['~/assets/style/label-style.css']`

Then, create this file  : frontend/assets/style/label-style.css :
with (you could change)  :

```
.annotation-text {
  font-size: 1.1rem !important;
  font-weight: 500;
  line-height: 2rem;
  font-family: 'Roboto', sans-serif !important;
  opacity: 0.6;
}
svg > g > g > g > g > text  {
  font-size: 0.9rem !important;
  font-weight: bold;
  font-style: italic;
  line-height: 2rem;
  font-family: sans-serif !important;
}
```

After saving the file, restart the Docker image and container creation. 
That works for sequence-labeling. 
HTH
